build.go

```diff
@@ -405,6 +405,26 @@ func (b *Builder) Build(s types.Storage, file string) error {
 			return err
 		}
 
+		if len(l.OverlayDirs) > 0 {
+			// get absolute paths for overlay_dirs source
+			overlayDirs, err := l.ParseOverlayDirs()
+			if err != nil {
+				return err
+			}
+
+			// copy overlay_dirs contents into a temporary dir in roots
+			err = copyImportsInRootfs(name, overlayDirs, opts.Config)
```

Comment:
I wonder if this and the related code can't all be moved into `overlay/`, since that's the only place it is used?

Reply:
You are right, I will move it there, thanks.